db/migrate/20160112190459_add_uuid_to_manifests.rb

```diff
@@ -0,0 +1,18 @@
+class AddUuidToManifests < ActiveRecord::Migration
+  def up
+    enable_extension 'uuid-ossp'
+    add_column :manifests, :uuid, :uuid
+
+    Manifest.all.each do |manifest|
```

Comment:
should we throw a quick class definition of `Manifest` in here for forward compatibility? http://railsguides.net/change-data-in-migrations-like-a-boss/